common/beerocks/bwl/include/bwl/mon_wlan_hal.h

```diff
@@ -55,7 +55,6 @@ class mon_wlan_hal : public virtual base_wlan_hal {
 
     virtual bool sta_channel_load_11k_request(const SStaChannelLoadRequest11k &req)      = 0;
     virtual bool sta_beacon_11k_request(const SBeaconRequest11k &req, int &dialog_token) = 0;
-    virtual bool sta_statistics_11k_request(const SStatisticsRequest11k &req)            = 0;
```

Comment:
Just squash this commit together with the three previous commits

Reply:
solved